packages/portfolio-contract/test/offer-shapes.test.ts

```diff
@@ -11,17 +11,15 @@ import {
   PortfolioStatusShapeExt,
 } from '../src/type-guards.ts';
 
-const usdc = withAmountUtils(makeIssuerKit('USDC'));
-const { brand: USDC } = usdc;
+const usdcKit = withAmountUtils(makeIssuerKit('USDC'));
+const usdc = usdcKit.make;
```

Comment:
why change the name of this? We usually bind the kit to the token name then use its utilities like usdc.unit

Reply:
this `usdc` was shadowed by another that was used a la `usdc(300n)` lots of times. I supposed it would be easier to review if fewer things were changed. Plus, lots of `usdc.make(300n)` would introduce linebreaks, which irks me.

maybe there's a better way to be consistent with typical use of `withAmountUtils`. dunno.